<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>330</y>
     <width>171</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>SETTING</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>115</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string> K:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="k">
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="startBtn">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>START</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="estimateBtn">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ESTIMATE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetBtn">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>RESET</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="finishBtn">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>FINISH</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>681</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ink Free</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 74, 3);
</string>
   </property>
   <property name="text">
    <string>k CENTRAL NODES ESTIMATION BY DYNAMIC PROGRAMMING</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
